### ISSUE TITLE
QA: Touch ID Error received

### ISSUE THREAD

**Issue by Tas4tari** [tari-project/wallet-ios#212]
**Description**
Can't set touch id. getting "there is no biometry" error. Unable to proceed 

**Device Details:**
iPhone 8

**Comment by Jasonvdb** [comment]
Another report: https://www.reddit.com/r/tari/comments/ga3xhh/aurora_app_bug/

**Comment by Anziskavds** [comment]
Tested on iOS. I removed my pin and fingerprints, downloaded the app again, and got the below two screens. It didn't go anywhere further than the last screenshot. After clicking on "Secure with pin" I receive "There is no biometry" and it just continues like this. I am unable to go any further from the "There is no biometry" screen. I had no issues after I have added a pin. I was able to set up my app completely after I added a pin. @Jasonvdb @Tas4tari @talmahaj Is this related to https://app.zenhub.com/workspaces/the-tari-ecosystem-5ccaecf0c55380274987d41b/issues/tari-project/wallet-ios/343?

![IMG_A28C8F0BA5FB-1.jpeg](https://images.zenhubusercontent.com/5cdd14f21f980125322a448a/db7ad351-7f86-4adc-92e1-093f09266204)

![IMG_889BB8FCFA0D-1.jpeg](https://images.zenhubusercontent.com/5cdd14f21f980125322a448a/9aade8e8-62e2-42ad-924d-643c4c4518ee)
